src/components/UserCreatedBounties/BountiesListing/BountiesTabItems.tsx

```
@@ -38,9 +42,33 @@ const BountiesTabItems: FC<IBountiesTabItemsProps> = (props) => {
 	];
 
 	const tabItems = bountyStatuses.map((status) => ({
-		children: <BountiesTable bounties={props.bounties?.length > 0 ? (props.bounties as IUserCreatedBounty[]) : []} />,
 		key: status.key,
-		label: <p>{status.label}</p>
+		label: <p>{status.label}</p>,
+		children:
```

ESLint

[error] Expected object keys to be in ascending order. 'children' should be before 'label'.

(sort-keys)